commons/file_manager.py

```diff
@@ -0,0 +1,109 @@
+#
+# @file_manager.py Copyright (c) 2020 Jalasoft.
+# 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+# 1376 subsuelo Edif. La Unión, Av. Gral. Inofuentes, Calacoto, La Paz, Bolivia
+# All rights reserved.
+#
+# This software is the confidential and proprietary information of
+# Jalasoft, ("Confidential Information"). You shall not
+# disclose such Confidential Information and shall use it only in
+# accordance with the termns of the license agreement you entered into
+# with Jalasoft.
+#
+# Author: Andres Cox
+# Version: 1.0
+
+import os
+from pathlib import Path
+from code_editor.orm_queries.orm_file import OrmFile
+from code_editor.orm_queries.orm_language import OrmLanguage
+from code_editor.orm_queries.orm_project import OrmProject
+from accounts.orm_queries.orm_user import OrmUser
+from commons.project_params import ProjectParameters
+from commons.settings import BASE_DIR
+
+# class file manager to modify local files
+
+
+class FileManager:
+
+    @staticmethod
+    def is_repeated_project(project_name, language_id, user_id):
+        user = OrmUser.get_user(user_id)
+        for project in OrmProject.get_all_projects(user):
+            print(project.name)
+            print(type(project.language.id), type(language_id))
+            print(project.language.id == language_id)
+            if (project.language.id == language_id) and (project.name == project_name):
+                return True
+        return False
+
+    # create a project in the media directory based in the language and user hashed folder
+    @staticmethod
+    def create_project(project_name, description, language_id, user_id):
+        user_dir = OrmUser.get_user_dir(user_id)
+        language = OrmLanguage.get_language(language_id)
+        project_path = 'media' / Path(user_dir) / language.name / \
+            language.version / project_name
+
+        # create project in the database and sends the id
+        print(FileManager.is_repeated_project(
+            project_name, language_id, user_id))
```

Comment:
print statement

Reply:
I deleted print statements in the last commit